frontend/website/static/js/leaderboard.js

```diff
@@ -60,5 +79,44 @@ function start() {
     );
 }
 
+function startChallenges() {
+  gapi.client
+    .init({
+      apiKey: "AIzaSyDIFwMr7SPGCLl_o6e4UZKi1q9l8snkUZs"
+    })
+    .then(function() {
+      return gapi.client.request({
+        path:
+          "https://sheets.googleapis.com/v4/spreadsheets/1CLX9DF7oFDWb1UiimQXgh_J6jO4fVLJEcEnPVAOfq24/values/Challenges!B:M"
+      });
+    })
+    .then(
+      function(response) {
+        const {
+          result: {
+            values,
+          }
+        } = response;
+
+        console.log(response);
+        const parentElem = document.getElementById("challenges-list");
+        
+        const latestChallenges = values[values.length - 1];
+      
+        for(var i = 0; i < latestChallenges.length; i+=2){
```

Comment:
What happens if someone forgets a description and the latestChallenges is an odd length?

Reply:
Change made in commit "Reformatting code" 0ebea61f6a105e2a2ef7c138847f147349c8af02